share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md

```diff
@@ -0,0 +1,51 @@
+# How to Create and Maintain a Tap
+Taps are external sources of Homebrew formulae. They can be created by anyone
```

Comment:
Possibly the world's smallest nit here - You can use taps for commands as well as/instead of formulae.